### PR TITLE
Motoraxis

### PR DESCRIPTION
**Pull request by pascalzauberzeug**
Use motoraxis endstop handling that works with CANOpen and Odrive Motors [now](https://github.com/zauberzeug/lizard/pull/39)

### CODE REVIEW

**Comment by rodja** [comment]
@pascalzauberzeug should we merge this into main? I would like to follow up with #65 because new changes are stacking up...

**Comment by pascalzauberzeug** [comment]
@rodja I think we can merge it when @angelom93 is back and had a look on it.
And we should not merge it without https://github.com/zauberzeug/field_friend/pull/65 since there are some fixes for this branch

File: field_friend/automations/plant_locator.py
[the remark this answers is not in the document]
Comment (pascalzauberzeug) [reply field_friend/automations/plant_locator.py:1]: Will be solved in https://github.com/zauberzeug/field_friend/pull/73

File: field_friend/vision/calibration.py
[the remark this answers is not in the document]
Comment (rodja) [reply field_friend/vision/calibration.py:1]: Yes I think the code can be removed. We can always get it from the history if we need to.

**Review by angelom93 — APPROVED** [review]

**Pull request merged**